worlds/sc2/docs/en_Custom Mission Orders.md

```diff
@@ -0,0 +1,536 @@
+          # layout is accessible
+          entrance: false
+          # Whether this mission is required to beat the layout
+          exit: false
+          # Whether this slot contains a mission at all
+          empty: false
+          # Which missions in the layout are unlocked by this mission
+          # This is normally set by the layout's type
+          next: []
+          # How many missions in the campaign must be beaten
+          # to access this mission
+          unlock_count: 0
+          # Which specific parts of the mission order must be beaten
+          # before this one can be accessed
+          unlock_specific: []
+          # Which missions are allowed to appear in this slot
+          # If not defined, the slot inherits the layout's pool
+          mission_pool:
+            - all missions
+          # Which specific difficulty this mission should have
+          difficulty: relative
+```
+## Access Rules
+
+If you've read the short option descriptions, you will have seen the word "access" a lot. Normally when you play a Starcraft 2 world, you have a table of missions in the Archipelago SC2 Client, and hovering over a mission tells you what missions are required to access it.
+
+This is still true for custom mission orders, but they also allow defining access rules for campaigns and layouts, which then apply to their entrance missions, and show up on those in the client. Entrance missions of layouts are those that either the layout type or your YAML mark with `entrance: true`. Entrance missions of campaigns are the entrance missions of layouts that have no access rules.
+
+Similarly, a mission in a custom mission order may require a campaign or layout to be beaten, which means beating all of their exit missions. Exit missions of layouts are those that either the layout type or your YAML mark with `exit: true`. Exit missions of campaigns are the exit missions of layouts that are marked as `required: true` and the missions your YAML marks with `required: true`. If you do not mark any required layouts or missions, all layouts of the highest order in the campaign will be required by default.
+
+There are four different ways to restrict access to a mission:
+- A layout's type will define the natural flow of missions, eg. to access a mission in a column layout you must beat the mission above it. Specific layout types are covered at the end of this document. You can override this flow using the `next` option on missions.
+- `order` and `unlock_count` allow defining a similar flow for layouts and campaigns. This process is explained below.
+- For missions, `unlock_count` defines how many other missions in the same campaign must be beaten before the mission can be accessed.
+- `unlock_specific` allows picking specific campaigns, layouts, or missions to turn into a requirement. This process is also explained below.
+
+## Shared options
+
+All the options below are listed with their defaults.
+
+---
```

Comment:
I recommend swapping all these horizontal lines to sub-headings to quickly identify / link to what option they're describing. So `### ``required`` option`, `### ``order`` option`, etc (double-backticks here should just be single-backticks, that's just me wrangling github's markdown formatter)